### PR TITLE
[id-ID.js] Fix some type and word

### PR DESCRIPTION
**Pull request by wellart**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description

Fixes #(issue)

## Type of change

Please delete any options that are not relevant.

- Translation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [ ] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

File: src/languages/id-ID.js [hunk 1]
```diff
@@ -379,8 +379,8 @@ export default {
     smtpDkimheaderFieldNames: "Header Keys untuk ditambahkan (Optional)",
     smtpDkimskipFields: "Header Keys not untuk ditambahkan (Optional)",
     wayToGetPagerDutyKey: "Anda dapat menambahkan melalui Service -> Service Directory -> (Select a service) -> Integrations -> Add integration. Lalu Anda dapat menjadi dengan kata kunci \"Events API V2\". Informasi tambahan {0}",
-    "Integration Key": "Integration Key",
-    "Integration URL": "Integration URL",
+    "Integration Key": "Kunci Integrasi",
+    "Integration URL": "Url Integrasi",
```
Comment (Computroniks) [inline src/languages/id-ID.js:383]:
```suggestion
    "Integration URL": "URL Integrasi",
```
URL should be capitalized probably as it is an acronym

File: src/languages/id-ID.js [hunk 2]
```diff
@@ -402,14 +402,14 @@ export default {
     enableProxyDescription: "Proxy berikut tidak akan berdampak ke monitor hingga diaktifkan. Anda dapat mengontrol menonaktifkan sementara proxy dari semua monitor dengan status aktivasi.",
     setAsDefaultProxyDescription: "Proxy berikut akan diaktifkan sebagai bawaan untuk monitor baru. Anda masih dapat menonaktifkan proxy secara terpisah untuk setiap monitor.",
     "Certificate Chain": "Certificate Chain",
-    Valid: "Sahih",
+    Valid: "Valid",
```
Comment (Computroniks) [inline src/languages/id-ID.js:405]: I don't speak Indonesian, is the English `Valid` more commonly used / more correct than the previous `Sahih`

File: src/languages/id-ID.js [hunk 3]
```diff
@@ -432,7 +432,7 @@ export default {
     User: "Pengguna",
     Installed: "Terpasang",
     "Not installed": "Tidak terpasang",
-    Running: "Berlari",
+    Running: "Berjalan",
```
Comment (Computroniks) [inline src/languages/id-ID.js:435]: From what I have looked up, this seems to mean walk over run, with the previous version meaning run?

File: src/languages/id-ID.js [hunk 4]
```diff
@@ -457,7 +457,7 @@ export default {
     cloudflareWebsite: "Situs Cloudflare",
     "Message:": "Pesan:",
     "Don't know how to get the token? Please read the guide:": "Tidak tahu cara mendapatkan token? Silakan baca panduannya:",
-    "The current connection may be lost if you are currently connecting via Cloudflare Tunnel. Are you sure want to stop it? Type your current password to confirm it.": "Koneksi saat ini mungkin hilang jika Anda saat ini terhubung melalui CloudflareTunnel. Apakah Anda yakin ingin menghentikannya? Ketik kata sandi Anda saat ini untuk mengonfirmasinya.",
+    "The current connection may be lost if you are currently connecting via Cloudflare Tunnel. Are you sure want to stop it? Type your current password to confirm it.": "Koneksi saat ini mungkin hilang jika Anda saat ini terhubung melalui Tunel Cloudflare. Apakah Anda yakin ingin menghentikannya? Ketik kata sandi Anda saat ini untuk mengonfirmasinya.",
```
Comment (Computroniks) [inline src/languages/id-ID.js:460]: We should probably keep this as it is as it is the name of a product

File: src/languages/id-ID.js (hunk 3 above)
Comment (wellart) [reply src/languages/id-ID.js:435]: run in term sport, is run, but in term of something start its walk

File: src/languages/id-ID.js (hunk 2 above)
Comment (wellart) [reply src/languages/id-ID.js:405]: yes valid is commonly used, sahih more arabic

File: src/languages/id-ID.js (hunk 4 above)
Comment (louislam) [reply src/languages/id-ID.js:460]: Unresolve conversation, I think `Cloudflare Tunnel` is a product name.

Comment (louislam) [reply src/languages/id-ID.js:460]: Sorry unresolve again.

**Pull request merged**